drivers/flash/flash_stm32_fmc_nand.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2025 Endress+Hauser GmbH+Co. KG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __ZEPHYR_DRIVERS_FLASH_STM32_FMC_NAND_H__
+#define __ZEPHYR_DRIVERS_FLASH_STM32_FMC_NAND_H__
```

Comment:
```suggestion
#ifndef __ZEPHYR_DRIVERS_FLASH_FLASH_STM32_FMC_NAND_H__
#define __ZEPHYR_DRIVERS_FLASH_FLASH_STM32_FMC_NAND_H__
```
to match header file path.